packages/mui-joy/src/Button/Button.tsx

```diff
@@ -178,21 +223,28 @@ Button.propTypes /* remove-proptypes */ = {
    * Either a string to use a HTML element or a component.
    * This is equivalent to `components.Root`. If both are provided, the `component` is used.
    */
-  component: PropTypes /* @typescript-to-proptypes-ignore */.elementType,
+  component: PropTypes.elementType,
   /**
    * If `true`, the button will take up the full width of its container.
    * @default false
    */
   fullWidth: PropTypes.bool,
+  /**
+   * The border-radius of the component.
+   * @default 'sm'
+   */
+  radius: PropTypes.oneOf(['lg', 'md', 'sm', 'xl', 'xs']),
+  /**
+   * The intensity of the shadow.
+   */
+  shadow: PropTypes.oneOf(['lg', 'md', 'sm', 'xl', 'xs']),
```

Comment:
```suggestion
  shadow: PropTypes.oneOf(['lg', 'md', 'sm', 'xl', 'xs']),
```
For a consistent design, and to minimise the props that need to be applied to every button, would this not be set in the theme? What's the use-case for this prop?

Reply:
I also have hard time to think about scenarios where apps would have different elevations.

Reply:
`shadow` prop is removed.